doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -157,14 +157,18 @@ compatible with the rest of |Coq|, up to a few discrepancies:
   (see :ref:`pattern_conditional_ssr`).  To use the
   generalized form, turn off the |SSR| Boolean ``if`` notation using the command:
   ``Close Scope boolean_if_scope``.
-+ The following two options can be unset to disable the incompatible
-  rewrite syntax and allow reserved identifiers to appear in scripts.
++ The following flags can be unset to make |SSR| more compatible with
+  parts of Coq:
 
-  .. coqtop:: in
+.. flag:: SsrRewrite
+
+   Controls whether the incompatible rewrite syntax is enabled (the default).
+   Disabling the flag makes the syntax compatible with other parts of Coq.
 
-     Unset SsrRewrite.
-     Unset SsrIdents.
+.. flag:: SsrIdents
 
+   Controls whether reserved identifiers can appear in scripts.  The default
```

Comment:
Hmm, I wasn't able to construct an example of this.  `Theorem exists: True.` is not accepted with either setting.  `Theorem Print: True.` is always accepted.  What would be an example?  I suppose it would be in an SSR-specific line (which I'm unfamiliar with)?  I might clarify the wording a bit.

Reply:
The use of the term "reserved identifier" is probably confusing because it may be understood as "keywords" when in fact, it designate the way that SSReflect names variables that were not given an explicit name:

```coq
Require Import ssreflect.

Unset SsrIdents.

Goal forall (P : Prop), P -> P.
move => *.
apply: _Hyp_.
(* Warning: The name _Hyp_ fits the _xxx_ format used for anonymous variables.
            Scripts with explicit references to anonymous variables are fragile. *)
(* Without the Unset SsrIdents above, this tactic would fail with:
            The identifier _Hyp_ is reserved. *)
```

Reply:
Got it, thanks.  "anonymous variables" is not a great term here because the "anonymous variables" are being assigned names.  So I used other language.  Please merge if agreeable.

I notice that `|Coq|` in the .rst file just becomes `Coq` in the final html document.  Does this construct still do something useful, e.g. uses a different font in the TeX version?  For `|SSR|` I see it replaces it with `SSReflect`.

`Small-scale reflection makes an extensive use of the programming
subset of |Gallina|, |Coq|’s logical specification language.`